internal/provider/tests_resource_test.go

```diff
@@ -21,13 +21,13 @@ resource "imagetest_tests" "foo" {
   driver = "k3s_in_docker"
 
   images = {
-    foo = "cgr.dev/chainguard/busybox:latest@sha256:98fa8044785ff59248ec9e5747bff259c6fe4b526ebb77d95d8a98ad958847dd"
+    foo = "cgr.dev/chainguard/busybox:latest@sha256:4559395ca443fc5d7be4dc813370ef3de4dda8561d1b9a8a24dc578027339791"
   }
```

Comment:
this image uses imagetest to build; so i cannot rebuild it; for usrmerge unless imagetest is published first.

So, should I be building it publically, and then updating this by hash?